ruby/ql/lib/codeql/ruby/dataflow/internal/FlowSummaryImplSpecific.qll

```diff
@@ -12,7 +12,7 @@ private import FlowSummaryImpl::Public
 private import codeql.ruby.dataflow.FlowSummary as FlowSummary
 
 /** Gets the parameter position of the instance parameter. */
-int instanceParameterPosition() { none() } // disables implicit summary flow to `self` for callbacks
+ArgumentPosition instanceParameterPosition() { none() } // disables implicit summary flow to `self` for callbacks
```

Comment:
I _think_ that this has the same explanation as my comment before, but it's a bit surprising to see `Gets the parameter position` in the QLDoc but `ArgumentPosition` as the result type.

Reply:
Yes, correct. I agree that the QL doc is not optimal.